src/main/scala/io/iohk/ethereum/validators/BlockHeaderValidator.scala

```diff
@@ -15,13 +15,19 @@ object BlockHeaderValidatorImpl {
   val GasLimitBoundDivisor: Int = 1024
   val MinGasLimit: BigInt = 5000 //Although the paper states this value is 125000, on the different clients 5000 is used
   val MaxGasLimit = Long.MaxValue // max gasLimit is equal 2^63-1 according to EIP106
+  val MaxPowCaches: Int = 2 // maximum number of epochs for which PoW cache is stored in memory
+
+  case class PowCacheData(cache: Array[Int], dagSize: Long)
 }
 
 class BlockHeaderValidatorImpl(blockchainConfig: BlockchainConfig) extends BlockHeaderValidator {
```

Comment:
Maybe I'm missing something but it seems another `BlockHeaderValidator` separate instance is [being created in the `OmmersValidator`](https://github.com/input-output-hk/mantis/blob/phase/daedalus/src/main/scala/io/iohk/ethereum/validators/OmmersValidator.scala#L28), won't each of them have to create their respective caches separately? This is up to discussion but maybe we could avoid having to create this duplicated caches.

Reply:
wow this is a good catch! I was pretty sure we were using only a single instance of validator, totally missed that ommers validator creates a new one. I'll fix it.